test/absorb-test.ts

```diff
@@ -64,7 +64,7 @@ describe('absorb', function () {
     expect(lU1.numAbsorbed).to.be.equal(0);
     expect(lU1.approxSpend).to.be.equal(0);
 
-    const usdcPrice = await priceFeeds['USDC'].price();
+    const [_, usdcPrice] = await priceFeeds['USDC'].latestRoundData();
```

Comment:
So we were accessing the stored price directly before? I guess `latestRoundData` is actually more correct since that's what we use in Comet and the official interface?

Reply:
yeah, i had made `price` a public variable, which was a mistake.

all the variables are now internal and you have to access them via `latestRoundData` or `getRoundData`, which are the functions defined by the `AggregatorV3Interface`